internal/engine/buildcontrol/docker_compose_build_and_deployer.go

```diff
@@ -141,6 +143,10 @@ func (bd *DockerComposeBuildAndDeployer) BuildAndDeploy(ctx context.Context, st
 		ps.EndPipelineStep(ctx)
 	}
 
+	var cluster v1alpha1.Cluster
+	// If the cluster fetch fails, that's OK.
+	_ = bd.ctrlClient.Get(ctx, ktypes.NamespacedName{Name: v1alpha1.ClusterNameDocker}, &cluster)
```

Comment:
What actually happens if this fails?

Reply:
I copied that from [image build and deployer](https://github.com/tilt-dev/tilt/blob/master/internal/engine/buildcontrol/image_build_and_deployer.go#L157-L158), it looks like currently the cluster is only used for platform detection/injection and the builder can continue without fetching the actual cluster info, but this may be more of an error condition in the future that needs to be handled. Do you want me to mark as such?